site/assets/scss/_component-examples.scss

```diff
@@ -275,12 +275,12 @@
 //
 
 .highlight {
-  padding: 1rem;
+  padding: 1.5rem;
   margin-bottom: 1rem;
   background-color: $gray-100;
 
   @include media-breakpoint-up(sm) {
-    padding: 1rem 1.5rem;
+    padding: 1.5rem;
```

Comment:
This is useless, you should drop the whole media-query since you're overriding the property with the exact same value.

Reply:
@ffoodd thanks, resolved in 013dba196d5d9253c23d8c1932ed92d5f1f206cd